scene/resources/bit_mask.cpp

```diff
@@ -58,8 +59,9 @@ void BitMap::create_from_image_alpha(const Ref<Image> &p_image) {
 
 		int bbyte = i / 8;
 		int bbit = i % 8;
-		if (r[i * 2])
+		if (r[i * 2 + 1] / 255.0 > p_threshold) {
```

Comment:
This doesn't immediately fill me with joy. create_from_image_alpha is exposed in through ClassDB and adding new (default) parameters to existing functions break GDNative ABI. 

I both don't like that it changes behavior even when p_threshold isn't passed and that this will break ABI. Can we get by without this change? If not I'd vastly prefer for bodge an extra method in under a different name for 3.0.

Reply:
I'll add a new method with the threshold parameter, keeping the original untouched so that it behaves completely as formerly. The new method will print a deprecation warning stating that in 3.1 you have the parameter in the original method.

I'm only dubious about the `+1`. Maybe @reduz can tell us if that indexing was broken in 3.0 so keeping the `+1` is backporting a fix.

Reply:
The `+ 1` was needed indeed. So, yes, a bugfix is being backported as well. Withoout it, it wasn't the alpha value the one being checked, but the luminance.

I've added a commit with the promised changes to the API to keep compatiblity.